ddtrace/bootstrap/sitecustomize.py

```diff
@@ -173,7 +173,7 @@ def _(threading):
         from ddtrace.appsec.iast._util import _is_python_version_supported
 
         if _is_python_version_supported():
-
+            # TODO: control any exception in this file
```

Comment:
Should this be done as part of this PR? Any chance that exceptions here might crash the user application?